tests/compat.rs

```diff
@@ -222,7 +222,7 @@ mod pgp {
                 "x86_64",
                 "spins round and round",
             )
-            .compression(CompressionType::Gzip)
+            .compression(CompressionType::Zstd)
```

Comment:
Just trying to increase coverage on the compat tests, we use Gzip already on one of them.